plugins/examples/change-logo/src/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { Headlamp, LogoProps, Plugin, Registry } from '@kinvolk/headlamp-plugin/lib';
```

Comment:
I am not very convinced with exporting a type that reads very generic at this level.
Since this is only used (as far as the user is concerned) at the plugin level, should we instead have it be available/exported from the Registry or the Plugin classes?